parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -50,6 +60,28 @@ installNvidiaContainerRuntime() {
     done
 }
 
+enableNvidiaPersistenceMode() {
```

Comment:
silly question. would it make sense to package this for mariner?

Reply:
Not at all, this is a great question! I actually looked into this and nvidia-persistenced is actually being [packaged](https://github.com/NVIDIA/yum-packaging-nvidia-persistenced/blob/main/nvidia-persistenced.spec) by the upstream. But when I checked the [source code](https://download.nvidia.com/XFree86/nvidia-persistenced/), looks like NVIDIA does not publish the source for each driver version, including the R525.85.12 that Azure Linux is supporting. 

Here's the [link](https://github.com/NVIDIA/nvidia-persistenced/issues/9#issuecomment-1555163280) to the github issue I filed to the upstream community regarding why source code for R525.85.12 is not released and the answer I got back. 

I also researched on the binaries/files shipped by the upstream nvidia-persistenced [package](https://rpmfind.net/linux/RPM/rpmfusion/nonfree/fedora/devel/x86_64/Packages/n/nvidia-persistenced-535.86.05-1.fc39.x86_64.html). The only instances that are related to actual functionality are **/usr/bin/nvidia-persistenced** and **/usr/lib/systemd/system/nvidia-persistenced.service** whereas the other files are either man pages or license files. /usr/bin/nvidia-persistenced is already shipped by the cuda rpm and thus the only thing left is the service file and since this file is not subject to changes in each release, I don't see a benefit of forcefully binding it in a package in Mariner unless we do it systematically like what the upstream does.

Reply:
@adithyaj thoughts on this?

Reply:
makes sense, I actually hit exactly the same issue :D I think the source from https://developer.download.nvidia.com/compute/cuda/redist/ _might_ have fabric manager source? but unsure.

not a blocker

Reply:
Yes, this link does contain the fabric manager source. This is actually where I grabbed the source to package fabric manager in Mariner. Unfortunately, I didn't see nvidia-persistenced source over there.

Reply:
d'oh, component confusion